typings/index.d.ts

```diff
@@ -1186,10 +1186,8 @@ type AwaitMessageCollectorOptionsParams<T extends MessageComponentType | Message
 
 export class Message extends Base {
   public constructor(client: Client, data: RawMessageData);
-  private _patch(data: RawPartialMessageData, partial: true): void;
-  private _patch(data: RawMessageData, partial?: boolean): void;
-  private _update(data: RawPartialMessageData, partial: true): Message;
-  private _update(data: RawMessageData, partial?: boolean): Message;
+  private _patch(data: RawPartialMessageData | RawMessageData): void;
+  private _update(data: RawPartialMessageData | RawMessageData): Message;
```

Comment:
Wasn't this function removed?

Reply:
It's still inherited from `Base`, which doesn't seem to have it in the types. I'll remove that line, not that it really matters.

Reply:
Ah I see, it should be moved there then